src/matter/interaction/InteractionProtocol.ts

```diff
@@ -32,8 +32,9 @@ export class ClusterServer<ClusterT extends ClusterSpec<any, any>> {
 
         // Create attributes
         for (const name in attributeDefs) {
-            const { id, template } = attributeDefs[name];
-            this.attributes[name as (keyof ClusterT["attributes"])] = new Attribute(id, name, template, attributesValues[name]);
+            const { id, template, defaultValueDef } = attributeDefs[name];
+            const defaultValue = attributesValues[name] !== undefined ? attributesValues[name] : defaultValueDef;
```

Comment:
attributesValues[name] cannot be undefined right, so the defaultValueDef will always be ignored...
As suggested in my previous comment, maybe you can ignore handling default values for now.

The correct way to handle them:

- AttributeValues should make optional attributes that have a default value
- you should check here for the value presence ( attributesValues.hasOwnProperty(name) ) and not if it is undefined since undefined can be a valid value (yes, undefined and not present are two different states in js...).

Reply:
Default stuff removed for now ...